debian/build.sh

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env bash
+
+trap 'umount "${MOUNT_DIR}" 2>/dev/null; rm -rf "${WORK_DIR}"' INT TERM EXIT
+
+# Prints a message in color
+function print_color() {
+    # Reset escape code
+    RST="\033[0m"
+    printf "\n%b%s%b\n" "${COLOR}" "${1}" "${RST}"
+}
+
+# Prints an error message in bold red then exits
+function die() {
+    COLOR="\033[01;31m" print_color "${1}"
+    exit "${2:-33}"
+}
+
+# Prints a warning message in bold yellow
+function warn() {
+    COLOR="\033[01;33m" print_color "${1}"
+}
+
+# The script requires root in several places, re-run the script with sudo if necessary
+function run_as_root() {
+    [[ ${EUID} -eq 0 ]] && return 0
+    warn "Script needs to be run as root, invoking sudo on script..."
+    echo
+    echo "$ exec sudo bash ${*}"
+    exec sudo PATH="${PATH}" bash "${@}"
+}
+
+# Get user inputs
+function get_parameters() {
+    DEBIAN=$(dirname "$(readlink -f "${0}")")
+
+    while ((${#})); do
+        case ${1} in
+            -a | --arch)
+                shift
+                case ${1} in
+                    arm64)
+                        DEB_ARCH=${1}
+                        OUR_ARCH=${DEB_ARCH}
+                        ;;
+                    arm)
+                        DEB_ARCH=armhf
+                        OUR_ARCH=${1}
+                        ;;
+                    ppc64le)
+                        DEB_ARCH=ppc64el
+                        OUR_ARCH=${1}
+                        ;;
+                    s390)
+                        DEB_ARCH=${1}x
```

Comment:
for s390 you append `x`, for arm you spell out `armhf`. Let's be consistent here.

Reply:
Thanks, good catch.